app/src/main/java/com/tvtracker/MainActivity.kt

```diff
@@ -208,22 +209,29 @@ class MainActivity : ComponentActivity() {
                 Column() {
                     Row(modifier = Modifier
                         .fillMaxWidth()
-                        .background(Color(0xFFE0E0E0)), horizontalArrangement = Arrangement.SpaceBetween) {
+                        .background(Color(0xFFE0E0E0)),
+                        horizontalArrangement = Arrangement.SpaceBetween) {
                         //Close Button
-                        Button( onClick = { openDialog = false}, colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent), elevation = ButtonDefaults.elevation(0.dp, 0.dp)) {
+                        Button( onClick = { openDialog = false},
+                            colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent),
+                            elevation = ButtonDefaults.elevation(0.dp, 0.dp),
+                            
+                            ) {
                             Icon(
                                 Icons.Outlined.Close,
-                                contentDescription = "Close",
+                                contentDescription = stringResource(R.string.Close),
```

Comment:
Converted to a string resource, loose strings are unsafe in the code labels and descriptions, and not converting is against best practices. 
https://developer.android.com/guide/topics/resources/string-resource#String

Reply:
Good.